polylogue/cli/app.py

```
@@ -547,7 +604,7 @@ def _prompt_branch_followups(ui, conversation, args, html_path: Optional[Path],
             if branch_choice:
                 _display_branch_diff_for_id(conversation, branch_choice, ui)
         elif choice.startswith("Write"):
-            target = _resolve_html_output_path(conversation, getattr(args, "html_out", None), False)
+            target = _resolve_html_output_path(conversation, getattr(args, "out", None), False)
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)